cranelift/wasm/src/environ/spec.rs

```diff
@@ -847,4 +847,28 @@ pub trait ModuleEnvironment<'data>: TargetEnvironment {
     fn wasm_features(&self) -> WasmFeatures {
         WasmFeatures::default()
     }
+
+    /// Indicates that this module will have `amount` submodules.
```

Comment:
So this only includes nested submodules, not the top level module? Good to make this explicit in the comment.

Reply:
Ah good point, but yes, this only includes the top-level module's modules (or whatever the current module is), basically just "we found the `module` section in the wasm file"